server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/RootNode.java

```diff
@@ -57,11 +57,15 @@ private void createCopyBookDefinitionNode(Multimap<String, Location> definition)
   }
 
   private List<SyntaxError> waitForVariableStructure() {
-    addProcessStep(this::addCopyNodesToTree);
+    addProcessStep(this::updateCopyNodesByPositionInTree);
     return ImmutableList.of();
   }
 
-  private List<SyntaxError> addCopyNodesToTree() {
+  /**
+   * Rearrange the copyBook Nodes in the syntax tree based on position
+   * @return List<SyntaxError>
+   */
+  public List<SyntaxError> updateCopyNodesByPositionInTree() {
```

Comment:
You may keep this method private and call the `process` method in tests instead.